.gitlab-ci.yml

```
@@ -26,23 +28,41 @@ variables:
     --time=00:10:00 
     --account=SHELLARD-SL3-GPU 
     --partition=ampere
+  FCOMPARE_FLAGS: >
+    --abs_tol 1e-10
+    --rel_tol 1e-10
+    --abort_if_not_all_found
+  LOCKED_FILENAME: >
+    file.lock
```

Just to explain the point of this...
```suggestion
  # We can only run a single job at a time with --qos=INTR
  QOS_INTR_LOCK_FILE: qos-intr.lock
```
P.S. there isn't a need to use the YAML line join `>` syntax since it's a single line!